controlpanel/frontend/static/javascripts/modules/role-names.js

```diff
@@ -0,0 +1,53 @@
+moj.Modules.roleNames = {
+  id: "role_name",
+  selectId: "role_name-select",
+  formClass: "appRoles",
+  roleListEndpoint: "/parameters/form/role-list.js",
+  roles: null,
+
+  init() {
+    if (document.querySelectorAll("." + this.formClass).length) {
```

Comment:
If you just set `formClass: ".appRoles"`, you don't need to do this ugly string concatenation.